sdk/identity-v2/azure-identity/src/main/java/com/azure/v2/identity/InteractiveBrowserCredential.java

```diff
@@ -140,8 +142,7 @@ public AuthenticationRecord authenticate(TokenRequestContext request) {
     public AuthenticationRecord authenticate() {
         String defaultScope = AzureAuthorityHosts.getDefaultScope(authorityHost);
         if (defaultScope == null) {
-            LoggingUtil.logCredentialUnavailableException(LOGGER, new CredentialUnavailableException(
-                "Authenticating in this " + "environment requires specifying a TokenRequestContext."));
+            LOGGER.atError().log("Authenticating in this environment requires specifying a TokenRequestContext.");
```

Comment:
Same: 

LoggingUtil.logCredentialUnavailableException

was also throwing the credential unavailable exception.
I know, not implied from the API, can be better.

We need to log and throw the exception here.